nest/src/channel/channel.gateway.ts

```diff
@@ -31,15 +34,35 @@ enum acknoledgementStatus {
 @WebSocketGateway(3333, {
   cors: {
     credentials: true,
+    origin: 'http://localhost:3000',
```

Comment:
That's good to strict the origin to a specific urls, but it's better to define them in the env file! smth like this :
```
    origin: [
      process.env.FRONTEND_URL,
      process.env.BACKEND_URL,
    ],
```

Reply:
Yeah it is always good to hide the origins !

Reply:
So it seems there is currently a bug with react-scripts version 5, where process is not defined. @williamollio do you know anything about that by chance?